dogs.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Page</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Neuton:ital@0;1&family=Work+Sans:ital,wght@0,300;0,600;1,300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="./styles/styles.css">
+  </head>
+  <body>
+    <a href="#main-content" class="hidden">Skip to main content</a>
+    <main>
+      <header>
+        <img src="./images/adoptadog-logo.svg" alt="Adoptadog logo" width="300">
+        <h2 class="green" style="font-style: italic">Seattle's premiere dog adoption agency</h2>
+        <img src="./images/cart-icon.svg" alt="shopping cart icon" width="25">
+        <p class="white">$1234.56</p>
+        <a href="./checkout.html"><p class="green-button">Checkout</p></a>
+      </header>
+      <nav style="background-color: #ececec">
+        <ul class="no-bullets">
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="./dogs.html">Dogs</a></li>
+          <li><a href="./blog.html">Blog</a></li>
+        </ul>
+      </nav>
+      <section>
+        <h1>Here are all of our Dogs!</h1>
+        <article>
```

Comment:
I'd argue these aren't articles. To me an article is an actual article that you read. I'd just use a div here myself. You'll also want some sort of container to go around the dog tiles (the index too) for layout